src/humanize/number.py

```diff
@@ -205,23 +205,44 @@ def intword(value: NumberOrString, format: str = "%.1f") -> str:
     except (TypeError, ValueError):
         return str(value)
 
+    is_negative = value < 0
+
+    if is_negative:
+        value *= -1
+
     if value < powers[0]:
-        return str(value)
+        return "-" + str(value) if is_negative else str(value)
     for ordinal, power in enumerate(powers[1:], 1):
         if value < power:
             chopped = value / float(powers[ordinal - 1])
             if float(format % chopped) == float(10**3):
                 chopped = value / float(powers[ordinal])
                 singular, plural = human_powers[ordinal]
+                if is_negative:
+                    return (
+                        "-"
+                        + " ".join(
+                            [format, _ngettext(singular, plural, math.ceil(chopped))]
+                        )
+                    ) % chopped
+
                 return (
                     " ".join([format, _ngettext(singular, plural, math.ceil(chopped))])
                 ) % chopped
```

Comment:
```suggestion
                return (
                    "-" if is_negative else ""
                    " ".join([format, _ngettext(singular, plural, math.ceil(chopped))])
                ) % chopped
```
I think something like this would work because strings within parenthesis are concatenated? It would reduce duplicated code. You could do something similar at every place you need to add a negative sign to the output.